sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -31,6 +32,13 @@ public partial class AuthorizationCodeCredential : Azure.Core.TokenCredential
         public override Azure.Core.AccessToken GetToken(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Core.AccessToken> GetTokenAsync(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public static partial class AzureAuthorityHosts
+    {
+        public static System.Uri AzureChinaCloud { get { throw null; } }
+        public static System.Uri AzureCloud { get { throw null; } }
+        public static System.Uri AzureGermanCloud { get { throw null; } }
```

Comment:
I thought we landed on
```suggestion
        public static System.Uri AzureChina { get { throw null; } }
        public static System.Uri AzurePublicCloud { get { throw null; } }
        public static System.Uri AzureGermany { get { throw null; } }
```

Reply:
Correct, I missed updating these, thanks!